### PR TITLE
add referral links to images in shop

### LINKED ISSUE #9918: Images should be made clickable on Shop page
Here is the link to the page: https://www.freecodecamp.com/shop

#### Issue Description

For better UI, I think this page would benefit from having links not only on Buttons, but also on images themselves. I've found myself repeatedly opening links in new tabs - to look at later, and to my surprise the images were opened, not the links, which was something I expected (learnt behaviour from other sites).

I wonder if I can make the change, if you decided it's needed :) Would love to help out!

### PR DESCRIPTION
**Pull request by dhcodes**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #9918 
#### Description

I replicated the behavior of all hrefs in [shop.jade](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/c1700a97930a356a29c874eee7c69cd7d9738a42/server/views/resources/shop.jade) to apply to all images in the shop too. Google Analytics should still reflect appropriately.

### CODE REVIEW

**Comment by BerkeleyTrue** [comment]
@dhcodes If you rebase of current staging it should pass that first security check.

**Comment by dhcodes** [comment]
@BerkeleyTrue okay, I think it's g2g now.

File: server/views/resources/shop.jade [hunk 1]
```diff
@@ -4,10 +4,13 @@ block content
     hr
     .row
         .col-xs-12.col-sm-10.col-sm-offset-1.col-md-8.col-md-offset-2.text-center
-            img.img-responsive.img-center(src='//i.imgur.com/FNFqn9Tg.jpg')
-            h3 Free Code Camp laptop stickers
-            h4 Get 2 for only $5, with free shipping anywhere!
-            p These durable 2" (5 cm) stickers sport a matte finish, and look great anywhere - especially on your laptop.
+          <form action="//www.paypal.com/cgi-bin/webscr" onSubmit="ga('send', 'event', 'Shop', 'clicked', 'Freecodecamp sticker');" method="post" target="_blank">
```
Comment (atjonathan) [inline server/views/resources/shop.jade]: Should we not be using the `pug` syntax for these?

Comment (BerkeleyTrue) [inline server/views/resources/shop.jade]: I think for now it's fine as long as it works. Where you able to verify this PR?

**Comment by atjonathan** [comment]
@BerkeleyTrue, from a visual and correct syntax/code point of view, it looks good to me. I'm pretty sure the google analytics will work but I am unable to test that.

**Comment by BerkeleyTrue** [comment]
@atjonathan Did you run the code and click the links?

File: server/views/resources/shop.jade [hunk 2]
```diff
@@ -29,15 +33,17 @@ block content
...
+              img.img-responsive.img-center(src='//i.imgur.com/BTtb8P0g.jpg')
             h3 Get $10 in credit for your own Digital Ocean server
             p Digital Ocean gives our community $25 in credit after you spend $25.
             a.btn.btn-cta.signup-btn.btn-block(href="//m.do.co/c/ba8a71773373" target='_blank' onClick="ga('send', 'event', 'Shop', 'clicked', '$10 Digital Ocean credit');") Claim your $10 credit
 
     hr
     .row
         .col-xs-12.col-sm-10.col-sm-offset-1.col-md-8.col-md-offset-2.text-center
-            img.img-responsive.img-center(src='http://i.imgur.com/YqZqTN6g.jpg')
+            a(href="//www.namecheap.com/?aff=102130" onClick="ga('send', 'event', 'Shop', 'clicked', 'Namecheap domain');")
```
Comment (atjonathan) [inline server/views/resources/shop.jade]: This link doesn't open in a new tab.

**Comment by atjonathan** [comment]
@dhcodes, please remove the large amount of spacing under the button add some between the button and "These durable"...

[https://gyazo.com/ec6f48a5b755af6eec183ddd37ffe209](https://gyazo.com/ec6f48a5b755af6eec183ddd37ffe209)

File: server/views/resources/shop.jade (hunk 2 above)
Comment (BerkeleyTrue) [inline server/views/resources/shop.jade]: It's not supposed too.

Comment (atjonathan) [inline server/views/resources/shop.jade]: @BerkeleyTrue, every other one does.

Comment (BerkeleyTrue) [inline server/views/resources/shop.jade]: Expect the name cheap link. It was discussed in the issue. Namecheap affiliate link does not work if it opens in a new tab.

**Comment by BerkeleyTrue** [comment]
@dhcodes Please remove the merge commit

**Comment by dhcodes** [comment]
Too fast @BerkeleyTrue. I'm working on it 👍

**Comment by dhcodes** [comment]
Okay, so I messed something up for a bit (still a bit dense on Git sometimes), but should be good now. To @atjonathan 's credit, I actually needed to convert the form to pug format anyway because when use HTML, it would generate a `<p></p>` that wasn't present in the file. Weird.

Anyway, I have tested this and it appears to compile correctly for Google Analytics and fixes the spacing issue @atjonathan referenced above. *fingers crossed*

**Comment by atjonathan** [comment]
@dhcodes awesome :tada:
I will test this and merge asap :+1:

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing to the most awesome open source community :tada:

**Pull request merged**